.github/workflows/docker.yml

```diff
@@ -80,7 +80,12 @@ jobs:
           echo $TAG
           docker build . -f docker/Dockerfile -t ${TAG} --build-arg CUDA_VERSION=${CUDA_VERSION}
           docker push $TAG
-      - name: Push docker image latest-cu12 as latest
+      - name: Build and push cu12 Docker image for Hopper GPU
+        if: endsWith(env.TAG, 'cu12') == true
```

Comment:
better to build for tag endswith 'latest-cu12' since the base image is `openmmlab/lmdeploy:latest-cu12` in Dockerfile_Hopper